### ISSUE TITLE
Should bypass functionality work with this blueprint?

### ISSUE THREAD

**Issue by Dinth** [ImSorryButWho/HomeAssistantNotes#23]
When im trying to arm an alarm with one of the sensors triggered, Ring keypad asks me to bypass but when i enter the code for the second time nothing happens. Should that functionality work?

**Comment by ImSorryButWho** [comment]
It should.  You shouldn't need to put the code in again - try just pushing the arm button again.

**Comment by Dinth** [comment]
Thanks for confirming. Are there any other prerequisites to make it work (for example in Alarmo configuration)?
When i enter the code and press Arm button, im hearing "Sensors require bypass" message from my Ring (v1). The confirmation "tick" button is blinking. If i press the configuration button the Ring just returns to unarmed state. If instead of pressing confirmation button i press arm button again i hear the "error sound", ring blinks red and the keypad returns to unarmed state

**Comment by ImSorryButWho** [comment]
Ahh, the v1 blueprint!  I don't actually have a v1 keypad to test with, so I've been relying on other's contributions.  I'll take a look later and see if I can figure it out.

**Comment by Dinth** [comment]
Yeah, thats what im doing with doors, but i would like bypass functionality for windows - get warned that im leaving a window open, but sometimes (leaving for a short while in the summer) be able to bypass that.

**Comment by TheRageOwns** [comment]
This is the only keypad (v1) that my wife likes and all of this is new to me, so this v1 blueprint has been a HUGE help to us; so, thank you. Has there been a solution to the bypass issue (above) though? That bypass issue is the only reason that I still need to have a tablet around for controlling the system (alarmo) just in case there is a issue. Thanks again!
